osschallenge/views.py

```diff
@@ -387,7 +393,11 @@ def RankingView(request):
 
     quarter = bisect.bisect(quarters, month)
     quarter_month = get_quarter_months(str(quarter))
-    contributors = User.objects.filter(profile__role_id=CONTRIBUTOR_ID)
+    try:
+        contributor_id = Group.objects.get(name="Contributor").id
+    except Group.DoesNotExist:
+        contributor_id = 1
+    contributors = User.objects.filter(groups__id=contributor_id)
```

Comment:
This code block can be shorted and you can simply do User.objects.filter(groups__name='Contributor')